server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -2566,6 +2567,61 @@ protected DiskOfferingVO createDiskOffering(final Long userId, final Long domain
             isCustomized = true;
         }
 
+        if (minIopsPerGb != null || maxIopsPerGb != null) {
+
+            if (!isCustomized) {
+               throw new InvalidParameterValueException("Cannot set Min/Max IOPS/GB for a fixed size disk offering");
```

Comment:
I did not understand this error here. Can you explain it a little further?

Reply:
You cannot have an IOPS/GB offering for a fixed sized disk. The whole Idea of an IOPS/GB type of offering makes sense when you have a custom offering so when you change the size of a disk, the IOPS also change

Reply:
So, for fixed size disks we cannot control the IOPS?  Why cannot we control/limit the IOPS of fixed size disks?

BTW, what do you understand as IOPS? input and output per second, meaning read and write speed.